.gitignore

```diff
@@ -1,4 +1,4 @@
 /vendor/
 /.idea/
 clover.xml
-.phpunit.result.cache
+.phpunit.result.cache
```

Comment:
👋🏻 This still appears to be a change that should not be in the PR. Could you make sure to not commit any changes to this file?